packages/arb-token-bridge-ui/src/hooks/useSelectedToken.ts

```diff
@@ -25,16 +25,62 @@ import {
   useTokensFromUser
 } from '../components/TransferPanel/TokenSearchUtils'
 import { useArbQueryParams } from './useArbQueryParams'
+import { ChainId } from '../types/ChainId'
 
-const commonUSDC = {
+const commonUSDC: ERC20BridgeToken = {
   name: 'USD Coin',
   type: TokenType.ERC20,
   symbol: 'USDC',
   decimals: 6,
-  listIds: new Set<string>()
+  listIds: new Set<string>(),
+  address: '',
+  logoURI:
+    'https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/ethereum/assets/0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48/logo.png'
 }
 
-export const useSelectedToken = () => {
+/**
+ * On ApeChain, if selectedToken is null, we default to ApeToken for regular transfers
+ * And constants.zero (ETH) for lifi transfers
+ */
+export function sanitizeSelectedTokenAddress({
+  sourceChainId,
+  destinationChainId,
+  erc20ParentAddress
+}: {
+  sourceChainId: number | undefined
+  destinationChainId: number | undefined
+  erc20ParentAddress: string | null
+}) {
+  /** Deposit to ApeChain from Ethereum, Superposition or Base is only supported through Lifi
+   *  We need to set the default token to ETH rather than ApeChain native token
+   *  For ArbitrumOne we default to native token (Ape)
+   */
+  if (!erc20ParentAddress && destinationChainId === ChainId.ApeChain) {
+    if (
+      sourceChainId === ChainId.Ethereum ||
+      sourceChainId === ChainId.Superposition ||
+      sourceChainId === ChainId.Base
+    ) {
+      return constants.AddressZero
+    }
+  }
+
+  /**
+   * For transfers from ApeChain, we default to ETH unless destination is ArbitrumOne
+   */
+  if (
+    !erc20ParentAddress &&
+    sourceChainId === ChainId.ApeChain &&
+    destinationChainId !== ChainId.ArbitrumOne
+  ) {
+    return constants.AddressZero
+  }
+}
```

Comment:
Don't we have to return `erc20ParentAddress` here?  I think then it would simplify `setSelectedToken` to:
```typescript
const sanitizedSelectedToken = sanitizeSelectedTokenAddress({
    sourceChainId: latestQuery.sourceChain,
    destinationChainId: latestQuery.destinationChain,
    erc20ParentAddress
})

return {
    token: sanitizeTokenAddress(sanitizedSelectedToken)
}
```